atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2185,9 +2185,8 @@ private boolean rollbackFailedTransactions(
             runTaskOnDeleteExecutor(kvs -> deleteCells(kvs, tableRef, keysToDelete));
         } catch (RejectedExecutionException rejected) {
             log.info(
-                    "Could not delete keys {} for table {}, because the delete executor's queue was full."
+                    "Could not delete keys for table {}, because the delete executor's queue was full."
                             + " Sweep should eventually clean these values.",
-                    UnsafeArg.of("keysToDelete", keysToDelete),
```

Comment:
Can we perhaps just list one or two or ten here, like we do with lock descriptors?

Reply:
at the very least, let us log the number of keys

Reply:
i doubt this will ever really be useful because we have a fallback mechanism i.e. is there an instance where we have actually using this for investigation?